### ISSUE TITLE
Manuals for development version missing from website

### ISSUE THREAD

**Issue by ninsbl** [OSGeo/grass-website#290]
Would be great to have manuals for the development version on the website.

Currently, latest manual version is 8.0:
https://grass.osgeo.org/learn/manuals/
Here, 8.0 is also flagged as preview which should be updated as well, I guess...

And https://grass.osgeo.org/grass-devel/manuals/index.html throws a 404 error...

**Comment by neteler** [comment]
> And https://grass.osgeo.org/grass-devel/manuals/index.html throws a 404 error...

Yes, because there is no cronjob/script yet to compile "main" in 
https://github.com/OSGeo/grass-addons/tree/grass8/utils/cronjobs_osgeo_lxd/

**Comment by neteler** [comment]
> > And https://grass.osgeo.org/grass-devel/manuals/index.html throws a 404 error...
> 
> Yes, because there is no cronjob/script yet to compile "main" in https://github.com/OSGeo/grass-addons/tree/grass8/utils/cronjobs_osgeo_lxd/

Addressed in   https://github.com/OSGeo/grass-addons/pull/719

**Comment by neteler** [comment]
Now the new cronjob is in place, please enjoy

https://grass.osgeo.org/grass-devel/manuals/

**Comment by neteler** [comment]
> Would be great to have manuals for the development version on the website.
> Currently, latest manual version is 8.0: https://grass.osgeo.org/learn/manuals/ 
> Here, 8.0 is also flagged as preview which should be updated as well, I guess...

This has been addressed in #291.

Remaining question: shall the development manual be listed as well at
https://grass.osgeo.org/learn/manuals/
?

**Comment by veroandreo** [comment]
> > Would be great to have manuals for the development version on the website.
> > Currently, latest manual version is 8.0: https://grass.osgeo.org/learn/manuals/
> > Here, 8.0 is also flagged as preview which should be updated as well, I guess...
> 
> This has been addressed in #291.
> 
> Remaining question: shall the development manual be listed as well at https://grass.osgeo.org/learn/manuals/ ?

We could do that, yes... I'd add them after the 7.8 link following the order in the dowload subpages, where preview version is the last one. Would that work out?

**Comment by neteler** [comment]
Wish implemented:

https://grass.osgeo.org/learn/manuals/
